packages/components/src/utils/test/strings.js

```diff
@@ -169,4 +169,14 @@ describe( 'normalizeTextString', () => {
 			'-'.repeat( dashCharacters.length )
 		);
 	} );
+
+	it( 'should normalize unicode to standard characters', () => {
+		expect( normalizeTextString( '①' ) ).toBe( '1' );
+		expect( normalizeTextString( 'Ⅸ' ) ).toBe( 'ix' );
+		expect( normalizeTextString( 'MC²' ) ).toBe( 'mc2' );
+		expect( normalizeTextString( 'ＰｌａｙＳｔａｔｉｏｎ　２' ) ).toBe(
+			'playstation 2'
+		);
+		expect( normalizeTextString( 'ＡＢＣ' ) ).toBe( 'abc' );
```

Comment:
It would be nice to add more cases here, and I'm open to suggestions.

Reply:
Maybe one for accent removal, which is part of the function? Like "Amélie".

Reply:
That's handled by the `removeAccents` library (props @tyxla), but I'm happy to add coverage.

Reply:
Yeah, we have a bunch of tests there, but having an extra test here can't hurt. I remember duplicating a bunch of WP core backend tests of `remove_accents` to ensure this works as expected.

Reply:
Thanks for confirmatio, @tyxla!